python/ray/data/dataset.py

```diff
@@ -3406,8 +3430,12 @@ def to_tf(
             label_column: Columns that correspond to model targets. If this is a
                 string, the target data is a tensor. If this is a list, the target data
                 is a ``dict`` that maps column names to their tensor representation.
-            prefetch_blocks: The number of blocks to prefetch ahead of the
-                current block during the scan.
+            prefetch_batches: The number of batches to fetch ahead of the current batch
+                to fetch. If set to greater than 0, a separate threadpool will be used
+                to fetch the objects to the local node, format the batches, and apply
+                the collate_fn. Defaults to 1. This is still an alpha API. You can
```

Comment:
Let's not document it as an alpha API (for those we'd call it experimental_prefetch_batches or something like that).

Reply:
We can say this replaces the legacy prefetch_blocks option, which you can use by setting the use_legacy... etc..

Reply:
updated